core/src/window_service.rs

```diff
@@ -171,26 +170,11 @@ fn verify_shred_slot(shred: &Shred, root: u64) -> bool {
 
 /// drop shreds that are from myself or not from the correct leader for the
 /// shred's slot
-pub(crate) fn should_retransmit_and_persist(
-    shred: &Shred,
-    bank: Option<Arc<Bank>>,
-    leader_schedule_cache: &LeaderScheduleCache,
-    my_pubkey: &Pubkey,
-    root: u64,
-) -> bool {
-    let slot_leader_pubkey = leader_schedule_cache.slot_leader_at(shred.slot(), bank.as_deref());
-    if let Some(leader_id) = slot_leader_pubkey {
-        if leader_id == *my_pubkey {
-            inc_new_counter_debug!("streamer-recv_window-circular_transmission", 1);
```

Comment:
given that this is a debug metric I assume we don't have data on this... should we collect info on this with log level info to see if we're seeing circular transmissions?